reladomo/src/main/java/com/gs/fw/common/mithra/cache/ReadWriteLockJdk21.java

```diff
@@ -0,0 +1,120 @@
+/*
```

Comment:
why is this necessary for JDK 21? Unsafe is not even deprecated in 21, let alone unavailable.
It gets deprecated in 23.

Additionally, this class is super critical to the correct functioning of the cache. The implementation below does not replace unsafe use, but totally rewrites the core functions. Putting this in production will result in severe issues for memory/cache heavy applications.

Reply:
Also, ReadWriteLock doesn't use unsafe in the first place.